pygaggle/data/convert_duot5_output_to_msmarco_run.py

```diff
@@ -0,0 +1,86 @@
+"""
+This script convert duoT5 output file to msmarco run file
+"""
+import argparse
+import collections
+import numpy as np
+from tqdm import tqdm
+
+parser = argparse.ArgumentParser()
+parser.add_argument("--t5_output", type=str, required=True,
+                    help="tsv file with two columns, <label> and <score>")
+parser.add_argument("--t5_output_ids", type=str, required=True,
+                    help="tsv file with five columns <query_id>, <doc_id_a>, <doc_id_b>, <rank_a> and <rank_b>")
+parser.add_argument("--mono_run", type=str, required=True,
```

Comment:
`input_run`  as it does not necessarily come from monoT5?

Reply:
true, will fix this too